### DISCUSSION TITLE
(csv) Export from ISO-TP Decoder....

### DISCUSSION THREAD

**Discussion by Mr-MIBonk** [collin80/SavvyCAN#829] [Q&A]
Hi, is there some trick to export the table result from ISO-TP Decoder into some file like CSV or copy the table into clipboard to paste it direct into Notepad oder excel for example?

![result](https://github.com/user-attachments/assets/dc30dde2-964e-4ef9-89cd-db7ecdda72da)

**Comment by collin80** [comment]
It... ah.. would probably have been a good idea to add an export function to that screen, huh? I'm not quite sure why I never added that.

**Comment by Mr-MIBonk** [comment]
Sorry for asking, but it would be very helpfull to get it short term for my CAN analysis, because i found no other Tool likes yours in Web with similar ISO-TP Function structure. Is some quick solution possible from your side?

**Reply by collin80** [reply]
Ok, there is a simple save format now. I can't say it's spectacularly great but it's something. I want to redo the whole window as it leaves a bit to be desired but at least it can save now. Check the upcoming continuous build and it will be in there.

**Comment by Mr-MIBonk** [comment]
Thank's a lot! But i think there is somesthing wrong... my original full CAN Log file is raound About 6MB, after iunselected some Frames the Export File is larger den 4GB(!) and i can't open it yet in Texteditor because it is to large. I think the different between 6MB input and 4GB output looks like some issue in export functionaly.
